### DISCUSSION TITLE
کار نکرد متد دیلیت مسیج بعد در صورت استفاده از آرگومان دیلیت افتر

### DISCUSSION THREAD

**Discussion by imanjahanyan** [kianahmadian/python-bale-bot#91] [Q&A]
با سلام و احترام
متاسفانه متوجه شدم که در صورت استفاده از آرگومان:
delete_after
در برخی موارد متد:
client.delete_message
کار نمی کنه
راه حلی پیشنهادی دارین؟

**Comment by kian-ahmadian** [comment]
سلام عزیز وقتتون بخیر
اگر امکان داره بخشی از کدتون که در حال استفاده از این متد هستید و ارسال کنید
